chapters/ch51.md

```diff
@@ -1,101 +1,102 @@
 > # 51. Choosing pipeline components: Task simplicity
 
---> _replace THIS LINE by your translation for the above line_
+# 51. Lựa chọn các thành phần cho pipeline: tính đơn giản của tác vụ
 
 > Other than data availability, you should also consider a second factor when picking components of a pipeline: How simple are the tasks solved by the individual components? You should try to choose pipeline components that are individually easy to build or learn. But what does it mean for a component to be "easy" to learn?
 
---> _replace THIS LINE by your translation for the above line_
+Ngoài sự sẵn có của dữ liệu, bạn cũng nên xem xét một nhân tố thứ hai khi lựa chọn các thành phần của một pipeline: Việc giải quyết từng tác vụ bằng các thành phần riêng lẻ đơn giản đến mức nào? Bạn nên cố gắng chọn những thành phần pipeline có thể dễ dàng xây dựng hay học riêng lẻ. Nhưng các thành phần "dễ" học nghĩa là gì?
 
 ![img](../imgs/C51_01.png)
 
 > Consider these machine learning tasks, listed in order of increasing difficulty:
 
---> _replace THIS LINE by your translation for the above line_
+Xét những tác vụ học máy này, được liệt kê theo thứ tự độ khó tăng dần:
 
 > 1. Classifying whether an image is overexposed (like the example above)
 
---> _replace THIS LINE by your translation for the above line_
+1. Phân loại một bức ảnh có bị phơi sáng quá mức hay không (như trong ví dụ trên)
 
 > 2. Classifying whether an image was taken indoor or outdoor
 
---> _replace THIS LINE by your translation for the above line_
+2. Phân loại một bức ảnh được chụp trong nhà hay ngoài trời
 
 > 3. Classifying whether an image contains a cat
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại một bức ảnh có chứa một con mèo hay không
 
 > 4. Classifying whether an image contains a cat with both black and white fur
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại liệu một bức ảnh chứa một con mèo khoang đen trắng hay không
 
 > 5. Classifying whether an image contains a Siamese cat (a particular breed of cat)
 
---> _replace THIS LINE by your translation for the above line_
+Phân loại liệu một bức ảnh có chứa một con mèo Siamese (tên một loại mèo)
```

Comment:
Mình có nên dịch từ Siamese cat -> mèo Xiêm không nhỉ. Mấy chương sau cũng có lấy ví dụ này nhiều.

Reply:
Ủa, anh không biết con này là mèo Xiêm. Để sau sửa lại vì mấy hình có con mèo này cũng giữ nguyên là mèo Siamese rồi. @rootonchair em giúp sửa lại tên con mèo này được không? Khi nào sửa xong anh sẽ thay toàn bộ. Cảm ơn em.

Reply:
@aivivn em cũng ko biết nó là mèo Xiêm. Để tối nay em mở PR sửa lại

Reply:
Đã fix ở #368